reframe/REFRAME_KNOWN_METHODS.md

```diff
@@ -203,6 +208,39 @@ Response:
 }}
 ```
 
+### FindProviders Blinded
+
+A message for finding nodes with interest in a given key using double hashing to blind the key being requested.
+
+```ipldsch
+    type FindHashedSHA256Request struct {

```

Comment:
What is the contingency for replacing SHA256 with other function in the future?
Creating a new message type?